<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DIALOG</class>
 <widget class="QDialog" name="DIALOG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CONTROL DE INCIDENCIAS</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <widget class="QRadioButton" name="radioButton_insertar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertar </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_modificar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_eliminar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>801</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <widget class="QLineEdit" name="lineEdit_pedidos">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unidades</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_descripcion">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>60</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cod_producto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripción</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fecha">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_error">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nº pedidos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Error</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_unidades">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_incidencia">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>INCIDENCIA :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_codproducto">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID registro</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_indice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripción</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cod_producto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_DESCRIPCION">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_producto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PRODUCTO :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PRODUCTO">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_obtener">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OBTENER REGISTRO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dd-mm-aaaa</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>12</number>
   </property>
   <widget class="QPushButton" name="pushButton_limpiar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ejecutar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>INSERTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salir">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modif">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>MODIFICAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_elim">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget_errores">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>891</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID_REGISTRO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>FECHA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>COD_PRODUCTO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DESCRIPCIÓN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ERROR</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>UNIDADES</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nº DE PEDIDOS</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_producto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>COD_PRODUCTO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DESCRIPCIÓN</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>391</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <widget class="QPushButton" name="pushButton_filtro">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar filtro</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fechafin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha inicial</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fechaini">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>fecha final</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graf_inc">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gráfico descripción</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graf_err">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Gráfico errores</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dd-mm-aaaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dd-mm-aaaa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exp_datfilt">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>140</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportar datos filtrados</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_cargarTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar tablas</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>909</x>
     <y>19</y>
     <width>371</width>
     <height>641</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exp_reg">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar tabla registros</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <widget class="QPushButton" name="pushButton_BD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BD_ERR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_BD</tabstop>
  <tabstop>radioButton_insertar</tabstop>
  <tabstop>radioButton_incidencia</tabstop>
  <tabstop>lineEdit_indice</tabstop>
  <tabstop>lineEdit_fecha</tabstop>
  <tabstop>lineEdit_codproducto</tabstop>
  <tabstop>lineEdit_descripcion</tabstop>
  <tabstop>comboBox_error</tabstop>
  <tabstop>lineEdit_unidades</tabstop>
  <tabstop>lineEdit_pedidos</tabstop>
  <tabstop>pushButton_ejecutar</tabstop>
  <tabstop>pushButton_cargarTab</tabstop>
  <tabstop>tableWidget_errores</tabstop>
  <tabstop>tableWidget_producto</tabstop>
  <tabstop>lineEdit_fechaini</tabstop>
  <tabstop>lineEdit_fechafin</tabstop>
  <tabstop>pushButton_filtro</tabstop>
  <tabstop>pushButton_graf_inc</tabstop>
  <tabstop>pushButton_graf_err</tabstop>
  <tabstop>radioButton_producto</tabstop>
  <tabstop>lineEdit_PRODUCTO</tabstop>
  <tabstop>lineEdit_DESCRIPCION</tabstop>
  <tabstop>pushButton_exp_reg</tabstop>
  <tabstop>radioButton_modificar</tabstop>
  <tabstop>radioButton_eliminar</tabstop>
  <tabstop>pushButton_obtener</tabstop>
  <tabstop>pushButton_limpiar</tabstop>
  <tabstop>pushButton_modif</tabstop>
  <tabstop>pushButton_elim</tabstop>
  <tabstop>pushButton_salir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
